code/modules/events/event_container.dm

```diff
@@ -145,9 +203,11 @@ var/global/list/severity_to_string = list(EVENT_LEVEL_MUNDANE = "Mundane", EVENT
 		new /datum/event_meta(EVENT_LEVEL_MUNDANE, "CCIA General Notice",		/datum/event/ccia_general_notice, 	300),
 		new /datum/event_meta(EVENT_LEVEL_MUNDANE, "Mundane Vermin Infestation",/datum/event/infestation, 			60,		list(ASSIGNMENT_JANITOR = 15, ASSIGNMENT_SECURITY = 15, ASSIGNMENT_MEDICAL = 15)),
 		new /datum/event_meta(EVENT_LEVEL_MUNDANE, "Drone Malfunction",			/datum/event/rogue_maint_drones,	10,		list(ASSIGNMENT_ENGINEER = 30)),
-		new /datum/event_meta(EVENT_LEVEL_MUNDANE, "Visitor", 					/datum/event/visitor,				50, is_one_shot = TRUE)
+		new /datum/event_meta(EVENT_LEVEL_MUNDANE, "Visitor", 					/datum/event/visitor,				50, TRUE)
```

Comment:
You need to take care when setting positional parameters.
The signature of the New() function for the /datum/even_meta is as follows:
```
/datum/event_meta/New(
var/event_severity,
var/event_name,
var/datum/event/type,
var/event_weight,
var/list/job_weights,
var/is_one_shot = 0,
var/min_event_weight = 0,
var/max_event_weight = 0,
var/list/excluded_roundtypes,
var/add_to_queue = TRUE)
```
Note the presence/location of the var/list/job_weights parameter.
With your change your are setting the job_weights parameter to true.

Reply:
fixed.